packages/lambda-xyz/src/tile.set.cache.ts

```diff
@@ -9,20 +9,104 @@ export function getTileSet(name: string, projection: Epsg): TileSet | undefined
     return TileSets.get(tileSetId);
 }
 
-export async function loadTileSet(req: LambdaContext, name: string, projection: Epsg): Promise<TileSet | null> {
-    req.set('tileSet', name);
-    req.set('projection', projection);
+export async function loadTileSet(name: string, projection: Epsg): Promise<TileSet | null> {
     let tileSet = getTileSet(name, projection);
     if (tileSet == null) {
         tileSet = new TileSet(name, projection);
         TileSets.set(tileSet.id, tileSet);
     }
-    req.timer.start('tileset:load');
     const loaded = await tileSet.load();
-    req.timer.end('tileset:load');
     if (!loaded) {
         TileSets.delete(tileSet.id);
         return null;
     }
     return tileSet;
 }
+
+function titleizeName(name: string): string {
+    return name.replace(/_0-/g, ' 0.').replace(/_/g, ' ');
+}
+
+function individualTileSet(parent: TileSet, image: TileSetRuleImagery): TileSet {
+    const id = image.imagery.id.slice(3);
```

Comment:
can we use `unprefix(image.imagery.id)` here? which explains why we are trimming a id by 3 characters.